SmartDeviceLink/SDLFunctionID.m

```diff
@@ -4,36 +4,93 @@
+                            @14: SDLNameSpeak,
+                            @15: SDLNameSetMediaClockTimer,
+                            @16: SDLNamePerformAudioPassThru,
+                            @17: SDLNameEndAudioPassThru,
+                            @18: SDLNameSubscribeButton,
+                            @19: SDLNameUnsubscribeButton,
+                            @20: SDLNameSubscribeVehicleData,
+                            @21: SDLNameUnsubscribeVehicleData,
+                            @22: SDLNameGetVehicleData,
+                            @23: SDLNameReadDid,
+                            @24: SDLNameGetDtcs,
+                            @25: SDLNameScrollableMessage,
+                            @26: SDLNameSlider,
+                            @27: SDLNameShowConstantTbt,
+                            @28: SDLNameAlertManeuver,
+                            @29: SDLNameUpdateTurnList,
+                            @30: SDLNameChangeRegistration,
+                            @31: SDLNameGenericResponse,
+                            @32: SDLNamePutFile,
+                            @33: SDLNameDeleteFile,
+                            @34: SDLNameListFiles,
+                            @35: SDLNameSetAppIcon,
+                            @36: SDLNameSetDisplayLayout,
+                            @37: SDLNameDiagnosticMessage,
+                            @38: SDLNameSystemRequest,
+                            @39: SDLNameSendLocation,
+                            @40: SDLNameDialNumber,
+                            @32768: SDLNameOnHmiStatus,
+                            @32769: SDLNameOnAppInterfaceUnregistered,
+                            @32770: SDLNameOnButtonEvent,
+                            @32771: SDLNameOnButtonPress,
+                            @32772: SDLNameOnVehicleData,
+                            @32773: SDLNameOnCommand,
+                            @32774: SDLNameOnTbtClientState,
+                            @32775: SDLNameOnDriverDistraction,
+                            @32776: SDLNameOnPermissionsChange,
+                            @32777: SDLNameOnAudioPassThru,
+                            @32778: SDLNameOnLanguageChange,
+                            @32779: SDLNameOnKeyboardInput,
+                            @32780: SDLNameOnTouchEvent,
+                            @32781: SDLNameOnSystemRequest,
+                            @32782: SDLNameOnHashChange,
+                            @65536: SDLNameEncodedSyncPData,
+                            @65537: SDLNameSyncPData,
+                            @98304: SDLNameOnEncodedSyncPData,
+                            @98305: SDLNameOnSyncPData
+                            };
+        }
+        self.functionIDs = functionIDs;
     }
     return self;
 }
 
-- (NSString *)getFunctionName:(int)functionID {
-    return [functionIDs objectForKey:[NSString stringWithFormat:@"%d", functionID]];
+- (SDLName)getFunctionName:(int)functionID {
```

Comment:
The `get` function naming is very Java. Should probably be more like `functionNameForId:`